### PR TITLE
Added a checkbox to notate VIP status

### PR DESCRIPTION
**Pull request by akemidx**
# Description
A checkbox has been added to a user profile that will allow that user to be notated as a VIP. This may be used mark special users who you wouldn't give elevated permissions to.

## Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?
Tested in local version of Snipe-IT

**Test Configuration**:
* PHP version:
* MySQL version
* Webserver version
* OS version

# Checklist:

- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [x] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/Http/Controllers/Api/UsersController.php [hunk 1]
```diff
@@ -246,6 +246,7 @@ public function index(Request $request)
                         'two_factor_optin',
                         'two_factor_enrolled',
                         'remote',
+                        'vipuser',
```
Comment (snipe) [inline app/Http/Controllers/Api/UsersController.php]: This is a trivial thing, but I'd prefer if we use `vip` as the field name so it reads a little cleaner.

**Comment by snipe** [comment]
This looks great, Jes - excellent work! I only have one tiny nit, and that's that I'd prefer the variable/field name to `vip` from `vipuser` .

**Comment by snipe** [comment]
(If you see the commit from me on this PR to resolve a conflict, it's because I had *just* pushed out some changes to the `$fillable` array - that's not your fault, you and I were just working on the same-ish lines of code at the same time.)

File: app/Http/Controllers/Users/UsersController.php [hunk 2]
```diff
@@ -271,6 +271,7 @@ public function update(SaveUserRequest $request, $id = null)
         $user->activated = $request->input('activated', 0);
         $user->zip = $request->input('zip', null);
         $user->remote = $request->input('remote', 0);
+        $user->vipuser = $request->input('vip', 0);
```
Comment (snipe) [inline app/Http/Controllers/Users/UsersController.php]: I think you might have missed one here.

`$user->vipuser = $request->input('vip', 0);`

should probably be:

`$user->vip = $request->input('vip', 0);`

**Comment by snipe** [comment]
@akemidx Ping! Can we wrap up the last few quick changes so I can merge this?

**Comment by akemidx** [comment]
I'm getting an error when I try to push that change. Think my branch and the one i'm pushing too are out of sync

**Pull request merged**